src/modules/launcher/Plugins/Microsoft.Plugin.Sys/Main.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Plugin.Sys.Win32;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Windows;
+using System.Windows.Interop;
+using Wox.Infrastructure;
+using Wox.Plugin;
+
+namespace Microsoft.Plugin.Sys
+{
+	public class Main : IPlugin, IPluginI18n
+	{
+		private PluginInitContext context;
+
+		#region DllImport
+
+		internal const int EWX_LOGOFF = 0x00000000;
+		internal const int EWX_SHUTDOWN = 0x00000001;
+		internal const int EWX_REBOOT = 0x00000002;
+		internal const int EWX_FORCE = 0x00000004;
+		internal const int EWX_POWEROFF = 0x00000008;
+		internal const int EWX_FORCEIFHUNG = 0x00000010;
+
+		#endregion
+
+		public void Init(PluginInitContext context)
+		{
+			this.context = context;
+		}
+
+		public List<Result> Query(Query query)
+		{
+            if (query == null)
+            {
+                throw new ArgumentNullException(paramName: nameof(query));
+            }
+
+            var commands = Commands();
+			var results = new List<Result>();
+
+            foreach (var c in commands)
+			{
+				var titleMatch = StringMatcher.FuzzySearch(query.Search, c.Title);
+				if (titleMatch.Score > 0)
+				{
+					c.Score = titleMatch.Score;
+                    c.SetTitleHighlightData(titleMatch.MatchData);
+					results.Add(c);
+				}
+			}
+			return results;
+		}
+
+		private List<Result> Commands()
+		{
+			var results = new List<Result>();
+            results.AddRange(new[]
+            {
+                new Result
+                {
+                    Title = "Shutdown",
```

Comment:
Are the "Title" strings showing in the UI? 
If they do, we need to move them to the resource file, if not, can you define const strings and add a comment the are strings that should not be localized?